src/Sulu/Bundle/AdminBundle/Resources/js/views/List/tests/List.test.js

```diff
@@ -126,8 +135,11 @@ test('Should unbind the query parameter and destroy the store on unmount', () =>
     };
 
     const list = mount(<List router={router} />);
-    expect(router.bindQuery).toBeCalledWith('page', undefined, '1');
-    expect(router.bindQuery).toBeCalledWith('locale', undefined);
+    const page = router.bindQuery.mock.calls[0][1];
+    const locale = router.bindQuery.mock.calls[1][1];
```

Comment:
Now you are missing the essential part :wink:

You should also check if these have the correct value:

```javascript
expect(page.get()).toBe(undefined);
expect(locale.get()).toBe(undefined);
```